pkgs/development/python-modules/softlayer/default.nix

```diff
@@ -11,7 +11,6 @@
   click,
   prettytable,
   prompt-toolkit,
-  ptable,
```

Comment:
nit: might make sense to split in 2 commits:
- `python3Packages.softlayer: remove unused ptable dependency`
- `python3Packages.ptable drop`

Nix changes look fine, and my closure checker only found softlayer as dependent. But lets wait for eval to make sure!

Reply:
Sounds good to me. Split the commit.